Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -32,6 +34,8 @@ public PostControllerTest(WebApplicationFactory<Startup> factory)
             _factory = factory;
         }
 
+
+
```

Comment:
why add these random, useless empty lines as a diff?

Reply:
erased blank lines. I think this was made by accident.